configure.ac

```diff
@@ -2,10 +2,10 @@ dnl Process this file with autoconf to produce a configure script.
 AC_INIT(udftools, 1.3, , , [https://github.com/pali/udftools/])
 AC_CONFIG_AUX_DIR(config)
 AM_CONFIG_HEADER(include/config.h:include/config.in)
-AM_INIT_AUTOMAKE
+AM_INIT_AUTOMAKE([subdir-objects])
 
 dnl Checks for programs.
-AC_PROG_CC
+AC_PROG_CC([clang gcc])
```

Comment:
Not needed. `cc` is standard program name present on all systems. People or distributions who use different compiler, they already specify it in `./configure` call phase.

Reply:
Clean-up was made here:  f2ce3c2